packages/react-components/src/components/MessageThread.tsx

```diff
@@ -67,6 +67,37 @@ const getLatestChatMessage = (messages: (ChatMessage | SystemMessage | CustomMes
   return undefined;
 };
 
+/**
+ * Get the latest participant number from the message array.
+ *
+ * @param messages
+ */
+const getLatestParticipantNum = (messages: (ChatMessage | SystemMessage | CustomMessage)[]): number => {
+  for (let i = messages.length - 1; i >= 0; i--) {
+    const message = messages[i];
+    if (message.messageType === 'chat') {
+      return message.messageThreadParticipantCount ? message.messageThreadParticipantCount : 1;
+    }
+  }
+  return 1;
+};
+
+/**
+ * Get the latest read number for my last sent message from the message array.
+ *
+ * @param messages
+ */
+const getLatestReadNum = (messages: (ChatMessage | SystemMessage | CustomMessage)[]): number => {
```

Comment:
Use Message type from packages\react-components\src\types\ChatMessage.ts instead

Reply:
Removed this function as it's not needed anymore